uptrain/framework/remote.py

```diff
@@ -84,6 +84,19 @@ def list_checksets(self, skip: int = 0, limit: int = 10):
         response = self.client.get(url, params=params)
         return response.json()
 
+    def add_experiment(self, name: str, checkset: CheckSet, experiment_args: ExperimentArgs):
```

Comment:
This function also needs to take the settings object.

Reply:
Done